torch/nn/modules/module.py

```diff
@@ -355,6 +356,7 @@ def _slow_forward(self, *input, **kwargs):
         return result
 
     def __call__(self, *input, **kwargs):
+        import torch.jit
```

Comment:
Having an import inside the `__call__` of `nn.Module` is an unacceptable overhead. It somehow worked before so it should be possible to avoid it

Reply:
ok, I'll look into this again then